lib/core/device.js

```diff
@@ -365,6 +365,8 @@ device.detectUsb = function(driverId, cb) {
                found = true;
              } else if (p.comName.match('/dev/tty.+')) {
                found = true;
+             } else if (p.comName.match('/dev/ttyUSB.+')) {
```

Comment:
I don't think this is necessary anymore, as the previous `tty.+` should match ttyUSB* as well?

Reply:
the '.+' is one character catch only, maybe with 'tty*' in the second if but I've not changed for this version because I will not touch on a code used for Mac platform (1 test for each platform I guess ?)

Reply:
How sorry, my regular expression's are a little bit false, I've tested without this line before and never work, I will retest with last version of the code

Reply:
Yeah, we used to match on `/dev/cu.+` for macOS, but this changed to `/dev/tty.+` (very recently) when we updated to a more recent version of the `node-serialport` library to make it work the same way as Linux.

Reply:
I see during november ! A good update with the node-serialport version 6.0.4 that solve an issue with linux too. I just tested the revert and the animas driver works well with just a oneliner to add linux platform in driver config, so my PR go to be useless ! :-)